profiles.yml

```diff
@@ -1,170 +1,208 @@
-     threshold: 0
-     branches:
-        subject: UPHENO:0001001
-        object: UPHENO:0001001
-     prefixes:
-        subject: hp
-        object: zp
-   - subset: hp-mp
-     method: semsimian
-     ontology: upheno1
-     threshold: 0.7
-     branches:
-        subject: UPHENO:0001001
-        object: UPHENO:0001001
-     prefixes:
-        subject: hp
-        object: mp
-   - subset: hp-hp
-     method: semsimian
-     ontology: upheno1
-     threshold: 0.7
-     branches:
-        subject: UPHENO:0001001
-        object: UPHENO:0001001
-     prefixes:
-        subject: hp
-        object: hp
-   - subset: hp-zp
-     method: semsimian
-     ontology: upheno1
-     threshold: 0.7
-     branches:
-        subject: UPHENO:0001001
-        object: UPHENO:0001001
-     prefixes:
-        subject: hp
-        object: zp
+  - subset: all
+    method: semsimian
+    ontology: upheno1
+    threshold: 0
+    branches:
+      subject: UPHENO:0001001
+      object: UPHENO:0001001
+  - subset: hp-mp
+    method: semsimian
+    threshold: 0
+    ontology: upheno1
+    branches:
+      subject: UPHENO:0001001
+      object: UPHENO:0001001
+    prefixes:
+      subject: hp
+      object: mp
+  - subset: hp-hp
+    method: semsimian
+    ontology: upheno1
+    threshold: 0
+    branches:
+      subject: UPHENO:0001001
+      object: UPHENO:0001001
+    prefixes:
+      subject: hp
+      object: hp
+  - subset: hp-zp
+    method: semsimian
+    ontology: upheno1
+    threshold: 0
+    branches:
+      subject: UPHENO:0001001
+      object: UPHENO:0001001
+    prefixes:
+      subject: hp
+      object: zp
+  - subset: hp-mp
+    method: semsimian
+    ontology: upheno1
+    threshold: 0.7
+    branches:
+      subject: UPHENO:0001001
+      object: UPHENO:0001001
+    prefixes:
+      subject: hp
+      object: mp
+  - subset: hp-hp
+    method: semsimian
+    ontology: upheno1
+    threshold: 0.7
+    branches:
+      subject: UPHENO:0001001
+      object: UPHENO:0001001
+    prefixes:
+      subject: hp
+      object: hp
+  - subset: hp-zp
+    method: semsimian
+    ontology: upheno1
+    threshold: 0.7
+    branches:
+      subject: UPHENO:0001001
+      object: UPHENO:0001001
+    prefixes:
+      subject: hp
+      object: zp
+  - subset: hp-fbcv
+    method: semsimian
+    ontology: upheno1
+    threshold: 0
+    branches:
+      subject: UPHENO:0001001
+      object: UPHENO:0001001
+    prefixes:
+      subject: hp
+      object: fbcv
+  - subset: hp-fbcv
+    method: semsimian
+    ontology: upheno2-lattice
+    threshold: 0
+    branches:
+      subject: UPHENO:0001001
+      object: UPHENO:0001001
+    prefixes:
+      subject: hp
+      object: fbcv
+  - subset: hp-hp
+    method: semsimian
+    ontology: phenio-monarch
+    threshold: 0
+    prefixes:
+      subject: hp
+      object: hp
+  - subset: hp-fbcv
+    method: semsimian
+    ontology: phenio-monarch
+    threshold: 0
+    prefixes:
+      subject: hp
+      object: fbcv
```

Comment:
Is it possible there is a problem because fbcv has to be spelled FBcv rather than FBCV?

Reply:
Maybe its more robust to declare branches.

Reply:
> Is it possible there is a problem because fbcv has to be spelled FBcv rather than FBCV?

We only use this for display purposes. The filtering process uses parent term property, and for this ontology is this one:

`parent_term: FBcv:0000000`

Reply:
This is the better one: FBcv:0001347

See here for more pheno configs: https://github.com/obophenotype/upheno-dev/blob/master/src/curation/upheno-config.yaml